src/include/pg_version_compat.h

```diff
@@ -123,37 +123,12 @@ RelationGetSmgr(Relation rel)
 #define ROLE_PG_READ_ALL_STATS DEFAULT_ROLE_READ_ALL_STATS
 #endif
 
-#define lnext_compat(l, r) lnext(l, r)
-#define list_delete_cell_compat(l, c, p) list_delete_cell(l, c)
-#define pg_plan_query_compat(p, q, c, b) pg_plan_query(p, q, c, b)
-#define planner_compat(p, c, b) planner(p, NULL, c, b)
-#define standard_planner_compat(a, c, d) standard_planner(a, NULL, c, d)
-#define GetSequencesOwnedByRelation(a) getOwnedSequences(a)
-#define GetSequencesOwnedByColumn(a, b) getOwnedSequences_internal(a, b, 0)
```

Comment:
Maybe we can leave `GetSequencesOwnedByRelation` and `GetSequencesOwnedByColumn` definitions for readability. `getOwnedSequences_internal` might be confusing. Although, currently it has only one use in the codebase, and the variable is defined such that it avoids confusion:
```SQL
List *columnOwnedSequences = getOwnedSequences_internal(relationId, attributeIndex + 1);
```
What do you think?

Reply:
I think we should merge this PR, and then in another PR we could introduce better function names -- in another header.

In the end, `getOwnedSequences_internal` is Postgres' function, and tbh I don't think we should use another name for it?

Reply:
Makes sense. Definitions like the following could also move to that header